### DISCUSSION TITLE
PROBLEMS IN INSTALLING MIKTEX

### DISCUSSION THREAD

**Discussion by gavrielsegre** [MiKTeX/miktex#1480] [Q&A]
Good morning

when I install Miktex and I try to open the Miktex Console I get the following error message:

"Impossible to find the point of entering SystemParametersInfoForDpi of the procedure in the library of the dynamic linking C:\Users\pc\AppData\Local\Programs\MIKTEX\miktex\bin\x64\Qt6Gui".

Could someone kindly help me?

**Comment by LLdaniel** [comment]
Are you running MikTeX on an old windows version? 
Could you please provide information about your exact Windows version?

Such error messages can arise when Qt6 is used on unsupported platforms. For Qt6 supported platforms see https://doc.qt.io/qt-6/supported-platforms.html#windows
